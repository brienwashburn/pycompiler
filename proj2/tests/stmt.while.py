while a: pass
else: bar

while a and b:
    pass
else:
    pass

while a and b: pass
else:
    pass

while a:
    a = 1
