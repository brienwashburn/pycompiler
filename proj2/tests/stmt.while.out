(Module
 (While (test (Name a)) (body (Pass)) (orelse (Expr (Name bar))))
 (While (test (BoolOp And (Name a) (Name b))) (body (Pass)) (orelse (Pass)))
 (While (test (BoolOp And (Name a) (Name b))) (body (Pass)) (orelse (Pass)))
 (While
  (test (Name a))
  (body (Assign (targets (Name a)) (value (Num 1))))
  (orelse)))
